DEVELOPING.md

```diff
@@ -71,29 +71,51 @@ To install `tilt` on PATH, run
 make install
 ```
 
-## Web UI
+## Performance
 
-`tilt` has an experimental web interface! We currently use it to run experiments.
+We use the built-in Go profiler to debug performance issues.
 
-By default, the web interface is off. To run it on port 8001:
+When `tilt` is running, press `ctrl-p` to start the profile, and `ctrl-p` to stop it.
+You should see output like:
 
 ```
-tilt up --port=8001
+starting pprof profile to tilt.profile
+stopped pprof profile to tilt.profile
+```
+
+This means that Tilt has successfully written profiling data to the file `tilt.profile`. Run this command:
+
+```
+go tool pprof tilt.profile
```

Comment:
maybe worth noting that this should be run from the directory where you started tilt?

Reply:
i like that! done